WS2012R2/lisa/xml/STOR_VHDX.xml

```diff
@@ -76,6 +76,7 @@
                 <param>DefaultSize=3TB</param>
                 <!-- <param>Path=G:\VHDs</param> -->
                 <param>fileSystems=(ext4 xfs btrfs)</param>
+                <param>ControllerType=SCSI</param>
```

Comment:
The controller type is already defined in the other param - SCSI/IDE=ID,port,sector_size,disk_size.
Why are we introducing a new parameter for ControllerType?